src/style.css

```diff
@@ -1 +1,302 @@
+  width: 100%;
+
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  padding-left: 33px;
+  box-sizing: border-box;
+}
+
+.big:focus {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.big:hover {
+  background-size: 19px;
+}
+
+.small:focus {
+  font-weight: bold;
+  background-size: 11px;
+  background-position: 13px center;
+  line-height: 22px;
+  text-align: left;
+  text-shadow: none;
+}
+
+.small:hover {
+  text-shadow: none;
+  background-size: 11px;
+  background-position: 13px center;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  font-weight: 300;
+  margin: 0;
+  padding: 0;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  outline: none;
+  color: #3d4e61;
+  background-image: url('./images/loop.svg');
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.big::placeholder {
+  color: #3d4e61;
+}
+
+.small::placeholder {
+  color: #3d4e61;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+  background-image: url('./images/loop.svg');
+  background-repeat: no-repeat;
+  background-position: 26px center;
+}
+
+.form:focus {
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: 26px center;
+  font-weight: bold;
+  line-height: 22px;
+  text-align: left;
+  text-shadow: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.big {
+  background-size: 19px;
+  background-position: 26px center;
+  height: 70px;
+  margin-top: 20px;
+  width: 100%;
+  top: 20px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+}
+
+.small {
+  font-size: 14px;
+  font-weight: 400;
+  background-image: url('./images/loop.svg');
+  background-size: 11px;
+  background-position: 13px center;
+  height: 42px;
+  width: 100%;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  padding-left: 33px;
+}
+
+.big:focus {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background-image: url('./images/loop.svg');
+  background-repeat: no-repeat;
+  background-position: 26px center;
+}
+
+.big:hover {
+  background-size: 19px;
+}
+
+.small:focus {
+  background-image: url('./images/loop.svg');
+  background-repeat: no-repeat;
+  font-weight: bold;
+  background-size: 11px;
+  background-position: 13px center;
+  line-height: 22px;
+  text-align: left;
+  text-shadow: none;
+}
+
+.small:hover {
+  text-shadow: none;
+  background-size: 11px;
+  background-position: 13px center;
+}
+*/
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  display: block;
+  margin-top: 20px;
+  outline: none;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  width: 100%; /* Розтягує форму на всю ширину контейнера */
+  box-sizing: border-box; /* Включає відступи та межі в ширину */
+}
+
+.big {
+  font-size: 16px;
+  background: url('/src/images/loop.svg') no-repeat;
```

Comment:
The path to the image seems incorrect. It should be relative to the CSS file. If the images directory is on the same level as the src directory, the path should be '../images/loop.svg'.